inst/stan/epinowcast.stan

```diff
@@ -118,6 +118,9 @@ data {
   int rep_rncol; 
   matrix[rep_fncol, rep_rncol + 1] rep_rdesign; 
   array[2, 1] real rep_beta_sd_p;
+  // Probability aggregation indicators
+  int agg_probs;
+  array[g, s] matrix agg_indicators;
```

Comment:
So this may have been my confusion but s is I think the number of unique snapshots regardless of group and t is reference time. 

I would expect this to throw a bunch of out of bounds errors for that reason?

I think we have a few options. 1. Use t instead and I think this will work and 2 store this by s (and not g and t) and then in `expected_obs_from_snap` we can use that we are looping over s to pull out the right matrix. Doing that might be a little tricky as you would need to construct the write list structure in the input (or collapse the user supplied one) so perhaps 1 is the way to go

Reply:
Ah, you found this even though I forgot to tag you! Okay, got it, I misunderstood. I think using t instead makes sense.